homework-02/my-calculator-v01/src/index.js

```diff
@@ -0,0 +1,19 @@
+function add(a, b) {
+    return (Number(a) + Number(b));
+}
+
+function sub(a, b) {
+    return Number(a) - Number(b);
+}
+
+function mul(a, b) {
+    return Number(a) * Number(b);
+}
+
+function div(a, b) {
+    if (Number(b) === 0) {
+        console.log('Cannot divide by zero use different second value');
```

Comment:
after this `console log` the division operation is made anyway.

to avoid this use `return`